HotelPriceScout/Data/Interface/IDashboard.cs

```diff
@@ -9,11 +9,11 @@ namespace HotelPriceScout.Data.Interface
 {
     public interface IDashboard
     {
-        List<WarningMessage> WarningMessages { get; set; }
-        bool BoolExceptionPopup { get; set; }
+        List<WarningMessage> WarningMessages { get; }
```

Comment:
Burde måske være IEnumerable, da det er en interface.

Reply:
Could be, men altså der skal jo tilføjes til listen, så det skal være en liste i Dashboard klassen.

Reply:
Så lad os beholde den sådan her, når vi ved der altid skal tilføjes til den.